src/main/resources/org/springframework/integration/kafka/config/spring-integration-kafka-2.1.xsd

```diff
@@ -188,6 +188,14 @@
 					<xsd:union memberTypes="listenerMode xsd:string"/>
 				</xsd:simpleType>
 			</xsd:attribute>
+			<xsd:attribute name="payload-type" type="xsd:string">
```

Comment:
Oh! Sorry, missed that reading from phone. Yes, agree with that condition. Even more: would be better to relax the rule at all: the schema is picked up for runtime from the class path anyway.